lib/puppet/ffi/posix.rb

```diff
@@ -0,0 +1,9 @@
+require 'ffi'
+
+module Puppet
+  module FFI
+    module Posix
```

Comment:
nit, can we upcase Posix? `lib/puppet/util/posix.rb` appears to use `Puppet::Util::POSIX`